ibis/expr/types/structs.py

```diff
@@ -112,6 +112,51 @@ def fields(self) -> Mapping[str, dt.DataType]:
         """Return a mapping from field name to field type of the struct."""
         return util.frozendict(self.type().pairs)
 
+    def lift(self) -> ir.Table:
+        """Project the fields of `self` into a table.
+
+        Existing fields from the child table are not retained in the
+        projection.
```

Comment:
I still think the phrase "child table" is confusing to users (or at least to myself) when thinking about tables and columns, rather than expressions. Feels like an implementation detail. I also wouldn't expect the fields from the containing table to be present, so mentioning it here feels unnecessary, I might just drop this whole sentence.

Reply:
Fair enough. I'll remove it.

Reply:
Done.